ament_index_python/ament_index_python/resources.py

```diff
@@ -135,7 +136,7 @@ def get_resource_types():
     return resource_types
 
 
-def has_resource(resource_type, resource_name):
+def has_resource(resource_type: str, resource_name: str) -> Union[str, os.PathLike[str], Literal[False]]:
```

Comment:
Now that we've changed `get_search_paths` to return a list of strings, this can be:

```suggestion
def has_resource(resource_type: str, resource_name: str) -> Union[str, Literal[False]]:
```